tests/examples/safe_remote_purchase/test_safe_remote_purchase.py

```diff
@@ -0,0 +1,64 @@
+#Test for Safe Remote Purchase (https://github.com/ethereum/solidity/blob/develop/docs/solidity-by-example.rst) ported to viper and optimized
+
+#Rundown of the transaction:
+#1. Seller posts item for sale and posts safety deposit of double the item value. Balance is 2*value.
+#(1.1. Seller can reclaim deposit and close the sale as long as nothing was purchased.)
+#2. Buyer purchases item (value) plus posts an additional safety deposit (Item value). Balance is 4*value
+#3. Seller ships item
+#4. Buyer confirms receiving the item. Buyer's deposit (value) is returned. Seller's deposit (2*value) + items value is returned. Balance is 0.
+import pytest
+from ethereum.tools import tester 
+from ethereum import utils
+
```

Comment:
Use `tests/setup_transaction_tests.py` to help with deployment (it already has `assert_tx_failed`).  I'd consider using `get_contract` as well.